core/trino-main/src/main/java/io/trino/operator/ValuesOperator.java

```diff
@@ -82,6 +82,7 @@ public OperatorContext getOperatorContext()
     }
 
     @Override
+    @SuppressWarnings("CheckReturnValue") // uses an Iterator helper to consume the remainder of the pages Iterator
```

Comment:
try something like
```
int ignore = Iterators.size(pages);
```

generally, it's highly recommended to avoid method-level suppressions

Reply:
This actually works! I replaced the other suppressions with this pattern as well.